### PR TITLE
Limit topic creation to hosts only

### PR DESCRIPTION
**Pull request by DaneEveritt**
Should prevent messes that occur when users make topics about a host they use which can cause duplicate topics, affiliate link spam, and so on.

Plus, if a user makes a topic, people can't get answers directly from the company.

### CODE REVIEW

File: rules/servers/rules.markdown [hunk 1]
```diff
@@ -103,7 +103,7 @@ IP to a server with no description or refer the user to a signature
 * No host may post in the topic of any other host. Host-to-host conversation must be taken to a PM
 * Hosts may offer giveaways in their topic, however the topic is not to be used for posting entries
 * Topics must pertain to Minecraft based hosting. Advertising non-Minecraft specific hosting is not allowed.
-
+* Topics in the Other Hosts section must be created by a representitive of a company. Members may not create a topic to advertise a host whom they use or recommend.
```
Comment (LordRalex) [inline rules/servers/rules.markdown]: representative, not representitive

File: rules/servers/rules.markdown [hunk 2]
```diff
@@ -103,7 +103,7 @@ IP to a server with no description or refer the user to a signature
 * No host may post in the topic of any other host. Host-to-host conversation must be taken to a PM
 * Hosts may offer giveaways in their topic, however the topic is not to be used for posting entries
 * Topics must pertain to Minecraft based hosting. Advertising non-Minecraft specific hosting is not allowed.
-
+* Topics in the Other Hosts section must be created by a representative of a company. Members may not create a topic to advertise a host whom they use or recommend.
```
Comment (LordRalex) [inline rules/servers/rules.markdown]: really no need to say Other Hosts, just make it apply to all since the main needs it anyway

Comment (DaneEveritt) [inline rules/servers/rules.markdown]: Isn't it kind of assumed/implied that if the topic made it into the main section it was made by a rep?

Comment (LordRalex) [inline rules/servers/rules.markdown]: Things should not be specific unless they need to be. This does not need to be specific to a section.

**Comment by JefeCC** [comment]
Never assume (assumptions make an...) or leave rules to "implication"

**Comment by LordRalex** [comment]
Please squash the commits into one

**Pull request closed without merging**